mkosi/__init__.py

```diff
@@ -7016,7 +6982,7 @@ def generate_secure_boot_key(args: CommandLineArguments) -> NoReturn:
                 )
             )
 
-    MkosiPrinter.print_step(f"Generating secure boot keys rsa:{keylength} for CN `{cn}`.")
+    MkosiPrinter.print_step(f"Generating secure boot keys rsa:{keylength} for CN {cn!r}.")
```

Comment:
Yeah, the comment was supposed to be half joking ;) I think it's better not to use backticks because they are special in the shell and make select-paste harder.

Reply:
I'm not disagreeing :) What I actually wanted to say is, that we should then remove them in the other cases as well.

Reply:
Oh! I pushed two more commits to remove the rest of the backticks.